.github/workflows/ci.yml

```diff
@@ -8,12 +8,13 @@ on:
 
 jobs:
   stack:
-    name: stack / ghc ${{ matrix.ghc }}
+    name: stack ${{ matrix.stack }} / ghc ${{ matrix.ghc }} / consul ${{ matrix.consul-version }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
         stack: ["2.3.1"]
         ghc: ["8.8.3"]
+        consul-version: ["1.3.1", "1.4.5", "1.5.3", "1.6.10", "1.7.11", "1.8.7", "1.9.0"]
```

Comment:
Might want to make this a multiline yaml list for reduced diffs

Reply:
Sure, I'll do that in my next pass.